### PR TITLE
No alert when find defective token

### PR DESCRIPTION
**Pull request by Veracity0**
I fired off a script and left my computer for a couple of hours.
When I returned, I found the script paused because KoLmafia just had to post a dialog box informing me I'd found a defective Game Grid token, requiring an "OK" confirmation.

Years ago, that might have been exciting. Once.

Now, it is, dare I say, annoying.

### CODE REVIEW

**Comment by gausie** [comment]
You could fire a desktop notification instead, which at least works on mac

**Comment by Veracity0** [comment]
Maybe. I'm not married to this change - although I will certainly keep it as a perma local change if not approved.

We used to pop up a dialog to show the text when you "used" a seal tooth, 15+ years ago. We removed it when it ceased to be interesting.

This is the only "alert" left of (what I see as) similar things.

If I'm alone st seeing it this way, I'll shrug and close this PR.

**Comment by gausie** [comment]
No no definitely merge this

**Comment by jaadams5** [comment]
Real life got in the way.  I was wondering whether "You could fire a desktop notification instead, which at least works on mac" meant there might be a suggested change and and wanted to confirm that there would be an acquisition message in the log but neither was intended to not approve.

**Review by jaadams5 — APPROVED** [review]

**Comment by Veracity0** [comment]
```
Visiting Jackass Plumber
You acquire an item: defective Game Grid token
```
The defective game grid token was introduced with the clan hot dog stand in July 2013; it unlocks the video game hot dog. Back then, it seemed worthwhile to alert the user when they found one, in case they wanted to unlock that hot dog.

These days, it's just not that interesting. My characters have found 15 of them. They do sell for 8.7 million + in the mall, but why select only that item for special notice? primitive alien {blowgun, loincloth, necklace, spear, totem} are vastly rarer; they sell for 50 million+ and there are from 2-6 of each - total - in the mall. My characters have actually found 16 such items. My Spacegate script does list them as special "goodies" found, but even without that, I'd not want an alert to pop up and pause the script.

tl;dr:
1) There is an acquistion message
2) The reason we originally had the alert popup has long since expired.
